test/test_treetensornetworks/test_solvers/test_dmrg.jl

```diff
@@ -139,7 +165,7 @@ end
     nsweeps,
     maxdim,
     cutoff,
-    outputlevel=2,
+    outputlevel=0,
```

Comment:
This was previously 2, so was generating a lot of output, and I don't think it was essential to this test so I turned it down to 0.